cmd/kube-controller-manager/app/plugins.go

```diff
@@ -43,7 +43,22 @@ func ProbeRecyclableVolumePlugins() []volume.VolumePlugin {
 	// The list of plugins to probe is decided by the kubelet binary, not
 	// by dynamic linking or other "magic".  Plugins will be analyzed and
 	// initialized later.
-	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins()...)
+	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins(defaultRecycler(60))...)
```

Comment:
Although that said, packaging the Creators and Deleters into the volume plugin makes sense to me.  They already have the cloud providers within them and the plugin itself seems like a logical place to go look when dealing with code that manipulates those volumes.

Design suggestions are welcome :)